### ISSUE TITLE
Create & Maintain handover file

### ISSUE THREAD

**Issue by vandagomes** [WeAreSnook/connected-camden#74]
## This figma file will make it have everything up to date and ready to build.

_Sometimes when exploring design and building at the same time, it's hard to understand what is a decision and what is an idea. Hopefully this will clarify and will help the team working in a more efficient way. This is an on-going task until the end of the project.

**Comment by HarryPotia** [comment]
@vandagomes yes this just sounds like good practice, don't think it can be broken down, but as a part of our overall documentation.

**Comment by mathewtrivett** [comment]
@HarryPotia I've added in the analytics plan as acceptance criteria
